utah-design-system-website/src/react/components/websiteContent/library/components/popups/banners/BannersDocumentation.jsx

```diff
@@ -30,7 +37,54 @@ function BannersDocumentation() {
       <h2 id="section-example">Example</h2>
       <StaticExample
         title="Small banners"
-        renderedExample={<LightBox image={bannersSmallScreenshot} alt="Small Banners" className="flex-4up-gap" />}
+        renderedExample={(
+          <div className="flex flex-col items-center">
+            <Banner
+              position="inline"
+              onClose={useCallback(() => showBanner({ message: 'banner closed' }), [showBanner])}
```

Comment:
"banner closed" would probably look better with a more verbose saying and proper punctuation and capitalization.
ie? "The banner's close button has been triggered. You are in charge of closing your own banners."